src/modules/situations/situation.module.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  display: flex;
+  padding: var(--spacings-medium);
+  border-radius: var(--border-radius-regular);
+}
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.linkText {
+  margin-top: var(--spacings-medium);
+}
+.title {
+  margin-bottom: var(--spacings-small);
+  padding: var(--spacings-small);
+}
+.body {
+  padding: var(--spacings-small);
+}
+.messageBox__button {
+  padding: var(--spacings-small);
+}
+.icon {
+  margin-top: var(--spacings-small);
+}
+
+.dialog {
+  padding: var(--spacings-large);
+  margin: auto auto;
+  border: none;
+  border-radius: var(--border-radius-circle);
+  background-color: var(--static-background-background_1-background);
+}
+
+.dialog::backdrop {
```

Comment:
TIL. Did not know that this was a pseudo selector.

Reply:
Jacob 🤝 🎉